app/components/SubscriptionScheduleClass.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+
+const SubscriptionSchedule = () => (
+    <div>
```

Comment:
We could later optimize this potentially by creating a checkbox component and passing the day in as a prop, so we don't have to repeat ourself. I can add that as an issue though for us to work on later - for now, looks good!